tools/common/Application.cs

```diff
@@ -794,6 +794,27 @@ public void InitializeCommon ()
 				}
 			}
 
+
+#if ENABLE_BITCODE_ON_IOS
+			if (Platform == ApplePlatform.iOS)
+				DeploymentTarget = new Version (9, 0);
+#endif
+
+			if (DeploymentTarget == null)
+				DeploymentTarget = Xamarin.SdkVersions.GetVersion (this);
```

Comment:
not a needed change, but you can do ??= 
```suggestion
				DeploymentTarget ??= Xamarin.SdkVersions.GetVersion (this);
```